.prettierignore

```diff
@@ -1,3 +1,4 @@
 .changeset/
 **/*.snap.js
 /docs/dist
+/docs/**/*.mdx
```

Comment:
Was prettier messing it up or?

Reply:
Yes, because it removes empty lines and for some features of expressive code need empty lines: https://expressive-code.com/key-features/text-markers/#adding-labels-to-line-markers